assets/scss/bootstrap/_wells.scss

```diff
@@ -0,0 +1,29 @@
+//
+// Wells
+// --------------------------------------------------
+
+
+// Base class
+.well {
+  min-height: 20px;
```

Comment:
Properties should be ordered background-color, border, border-radius, margin-bottom, min-height, padding